stock_available_to_promise_release/views/res_config_settings.xml

```diff
@@ -28,6 +28,26 @@
                             </div>
                         </div>
                     </div>
+                    <div class="col-12 col-lg-6 o_setting_box">
+                        <div class="o_setting_right_pane">
+                            <label for="stock_release_max_prep_time" />
+                            <div class="text-muted">
+                                When your release transfers, their expected date
+                                is rescheduled to now + this preparation time.
```

Comment:
```suggestion
                                is rescheduled to now + this preparation time (in minutes).
```

Reply:
It's displayed "in minutes" next to the input field, so is it useful?

Reply:
updated anyway